packages/yoga/src/heading/web/RightButton.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '../../Button';
+
+const RightButton = ({ onClick, icon }) => (
```

Comment:
What do you think about leaving the button position implicit? That way we can change the style in the future without interfering with the semantics of the API.
```suggestion
const Button = ({ onClick, icon }) => (
```

I think we should also pass the other props to button.